commands.json

```diff
@@ -6985,6 +7001,32 @@
             "stale"
         ]
     },
+    "LATENCY HISTOGRAM": {
+        "summary": "Return the cumulative distribution of latencies of a subset of commands or all.",
+        "since": "7.0.0",
+        "group": "server",
+        "complexity": "O(N) where N is the number of commands with latency information being retrieved.",
+        "acl_categories": [
+            "@admin",
+            "@slow",
+            "@dangerous"
```

Comment:
dangerous? Why?
I know this flag is a mess and it's unclear when to use it. My personal guideline is that a command is _dangerous_ if we want to be prompted by _redis-cli_ "Are you sure you want to run this [y/N]?" after we press ENTER. And by this standard I see no problem with this command.

Reply:
this acl **category** is automatically added when the `**admin**` flag is set.

the dangerous category is now defined:
> Potentially dangerous commands (each should be considered with care for various reasons).

i.e. any command that could have some reason to be considered dengarous, is flagged, and users needs to enable them one by one if they want it exposed (e.g. FLUSHDB).

Reply:
given this is inherited no change here correct @oranagra @yoav-steinberg?

Reply:
yes. it suxx but that's how it is.
the LATENCY command was always tagged as an `admin` command.